automation_tools/__init__.py

```diff
@@ -1910,6 +1941,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             sat_version=satellite_version,
             host=host
         )
+    execute(setup_code_coverage, host=host)
```

Comment:
Will the services be restarted after this? They need to be restarted otherwise coverage data collection won't work.